### ISSUE TITLE
Integrate microx viewer into qira

### ISSUE THREAD

**Issue by dbrumley** [BinaryAnalysisPlatform/bap#547]
@rvantonder has written a nice POC plugin at https://github.com/BinaryAnalysisPlatform/bap-plugins/tree/master/toy-debugger for viewing microx.  Conceptually it seems viewing an execution, whether real or microx, is a job for qira.

**Comment by rvantonder** [comment]
To add to this: I spent roughly an afternoon at some point looking at static2 in qira for possible integration, with graphs as a starting point. What I found was that qira's graph rendering is produced in a very roundabout way. Basically, create a dot file with the current vertices rendering, send it to the backend to get a visual layout of nodes using dot, and then parse the returned dot file and redraw the arrows/edges/vertices. Relevant code:

https://github.com/BinaryAnalysisPlatform/qira/blob/master/web/client/static/graph.js#L93
https://github.com/BinaryAnalysisPlatform/qira/blob/master/middleware/qira_webstatic.py#L73

The implication is that rendering execution through the trace at each step needs more thinking beyond the current rendering logic qira uses. Perhaps having a button/switch in qira to select an alternative renderer implementation for BIL/BIR makes sense.

Also remember that microx potentially operates over multiple traces; this would also have to be reflected in qira. There is currently no standard trace format (the debugger uses sets of csv files) for microx to serve as input to qira. Qira would need a new trace parser for microx (@nedwill pointed to https://github.com/BinaryAnalysisPlatform/qira/blob/master/tracers/angr/angr_trace.py as a potential reference some time ago).
